css/style.css

```diff
@@ -1,55 +1,58 @@
-html {
-    font-size: 120%;
-    margin: 60px;
-  }
+body {
+  font-size: 120%;
+  margin: 60px;
+  margin: 0;
+  padding: 0;
+  text-align: center;
+  font-family: cursive;
+}
+
+footer {
+  text-align: left;
+  background-color: pink;
+  border: 3px solid purple;
+  border-radius: 5px;
+}
+
+/* fonts */
+
+h1 {
+  color: mediumorchid;
+  font-family: cursive;
+  font-size: 60px;
+}
+
+h2 { 
+  color: mediumorchid;
+  font-family: cursive;
+}
+
+section {
```

Comment:
You can wrap all of the parts of the page in section elements with share styles for consistency